main/sw/inc/shellio.hxx

```diff
@@ -106,11 +105,11 @@ public:
 };
 
 /**************** SwReader/Reader ************************/
-// Basisklasse der moeglichen Optionen fuer einen speziellen Reader
+// base class for different specialized readers
 class Reader;
-// Ruft den Reader mit seinen Optionen, Dokument, Cursor etc.
+//  calls a document with its options, document, cursor etc.
```

Comment:
should it this call a Reader instead? It makes sense if "Reader" is the class

Reply:
yea, you are right. I fix this